src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -509,25 +509,26 @@ public void createCaseConversionNote(String caseUUIDString, String stageUUIDStri
 
     private UUID deriveTeamUUID(String caseUUIDString, String stageTypeString, String allocationTeamString) {
         UUID teamUUID;
-        if (StringUtils.isEmpty(allocationTeamString)) {
+        if (!StringUtils.hasText(allocationTeamString)) {
```

Comment:
Is it worth logging out this allocationTeamString as well?

Reply:
In this one... that if block is only called if the `allocatedTeamString` is `null` anyway and in both the subsequent log output it logs the resolved teamUUID, specifically the new final info level one. From the logic there's no way if can escape that method as null, if the `infoClient` call fails it should trigger the global handling anyway.